### PR TITLE
Client error handling

### PR DESCRIPTION
**Pull request by dmfalke**
Updates `ContactUs` UI and Service to support a `context` object that can supplement user messages.

Depends on https://VEuPathDB/WDKClient/pull/18

### CODE REVIEW

File: Model/src/main/java/org/eupathdb/common/model/contact/ContactUsParams.java [hunk 1]
```diff
@@ -18,12 +19,14 @@ public ContactUsParams(
       String referrer,
       String[] ccEmails,
       String message,
+      String context,
```
Comment (jtlong3rd) [inline Model/src/main/java/org/eupathdb/common/model/contact/ContactUsParams.java:22]: This update to the `ContactUsParams` constructor breaks two little-known unit tests. We should fix them before merging.

- [ ] Model/src/test/java/org/eupathdb/common/model/contact/ContactUsParamsTest.java
- [ ] Model/src/test/java/org/eupathdb/common/model/contact/ContactUsSubmitterTest.java

Comment (dmfalke) [reply Model/src/main/java/org/eupathdb/common/model/contact/ContactUsParams.java:22]: Ah, I will fix this. When do these tests run?

Comment (jtlong3rd) [reply Model/src/main/java/org/eupathdb/common/model/contact/ContactUsParams.java:22]: I don't believe these test are parts of our CI, so fixing them is definitely more of a nitpick.

Comment (dmfalke) [reply Model/src/main/java/org/eupathdb/common/model/contact/ContactUsParams.java:22]: I just pushed a fix for this

**Pull request merged**